packages/terra-menu/src/Menu.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popup from 'terra-popup';
+import SlideGroup from 'terra-slide-group';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import MenuItem from './MenuItem';
+import MenuItemGroup from './MenuItemGroup';
+import MenuDivider from './MenuDivider';
+import MenuContent from './_MenuContent';
+import MenuWidths from './_MenuWidths';
+import MenuUtils from './_MenuUtils';
+import styles from './Menu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * List of Menu.Item(s)/Menu.ItemGroup(s)/Menu.Divider(s) to be displayed as content within the Menu.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Callback function indicating a close condition was met, should be combined with isOpen for state management.
+   */
+  onRequestClose: PropTypes.func.isRequired,
+  /**
+   * Target element for the menu to anchor to.
+   */
+  targetRef: PropTypes.func.isRequired,
+  /**
+   * Bounding container for the menu, will use window if no value provided.
+   */
+  boundingRef: PropTypes.func,
+  /**
+   * CSS classnames that are append to the arrow.
+   */
+  classNameArrow: PropTypes.string,
+  /**
+   * CSS classnames that are append to the menu content inner.
+   */
+  classNameContent: PropTypes.string,
+  /**
+   * CSS classnames that are append to the overlay.
+   */
+  classNameOverlay: PropTypes.string,
+  /**
+   * Should the menu be presented as open.
+   */
+  isOpen: PropTypes.bool,
+  /**
+   * A string representation of the width in px, limited to:
+   * 160, 240, 320, 640, 960, 1280, 1760
+   */
+  contentWidth: PropTypes.oneOf(Object.keys(MenuWidths)),
```

Comment:
In the propsTable markdown, this proptype is undefined. I wonder if this is something we log as an issue with propsTable or if we just want to write out `PropTypes.oneOf(160, 240, 320, 640, 960, 1280, 1760)`

Reply:
Lets write it out as you mentioned @emilyrohrbough. That matches what we did in the `<Text />` component as well: https://github.com/cerner/terra-core/blob/master/packages/terra-text/src/Text.jsx#L29